tests/runfiles/linux.run

```diff
@@ -536,16 +536,13 @@ tests = ['refreserv_001_pos', 'refreserv_002_pos', 'refreserv_003_pos',
 tests = ['replacement_001_pos', 'replacement_002_pos', 'replacement_003_pos']
 
 # DISABLED:
```

Comment:
Go ahead and remove this disabled comment as well now that all of these tests are enabled.